client/common/sass/components/_filters.sass

```diff
@@ -69,3 +77,54 @@
 	margin: 0
 	font-size: 1.25rem
 	font-weight: 400
+
+
```

Comment:
I'm guessing this double line break is the cause of the lint issue and it's just giving the wrong line number?

Reply:
Seems like this was it! Thanks!